jenkins_pipelines/environments/common/pipeline-pull-request-envs.groovy

```diff
@@ -23,8 +50,40 @@ if (env.JOB_NAME == "uyuni-prs-ci-tests-jordi") {
     first_env = 9;
     last_env = 9;
     additional_repo_url = "http://minima-mirror-ci-bv.mgr.prv.suse.net/pull-request-repositories/reference_job_additional_repo";
-} else { //regular ci test
+} else if (env.JOB_NAME == "uyuni-prs-ci-tests") {
     first_env = 1;
     last_env = 4;
+} else if (env.JOB_NAME == "suma43-prs-ci-tests") {
+    first_env =5;
+    last_env = 6;
+    // spacewalk suma43 settings
+    // TODO: what happens if we change the environment? Will it break the environment??
+    server_release_package = '000product:sle-module-suse-manager-server-release'
+    proxy_release_package = '000product:sle-module-suse-manager-proxy-release'
+    pull_request_repo = 'https://github.com/SUSE/spacewalk.git'
+    builder_api = 'https://api.suse.de'
+    build_url = 'https://build.suse.de'
+    builder_project = 'Devel:Galaxy:Manager:4.3:PR'
+    source_project = 'Devel:Galaxy:Manager:4.3'
+    sumaform_tools_project = 'openSUSE.org:systemsmanagement:sumaform:tools'
+    test_packages_project = 'openSUSE.org:systemsmanagement:Uyuni:Test-Packages:Pool'
+    other_project = 'Devel:Galaxy:Manager:Head:Other'
+    el_client_repo = "${source_project}:EL9-SUSE-Manager-Tools"
+    EL_9 = 'SUSE_EL-9_Update_standard'
+    sles_client_repo = "${source_project}:SLE15-SUSE-Manager-Tools"
+    openSUSE_client_repo = "openSUSE.org:systemsmanagement:Uyuni:Master:openSUSE_Leap_15-Uyuni-Client-Tools"
+    ubuntu_client_repo = "${source_project}:Ubuntu22.04-SUSE-Manager-Tools"
+    build_repo = 'SLE_15_SP4'
+    other_build_repo = 'openSUSE_Leap_15.4'
+    url_prefix="https://ci.suse.de/view/Manager/view/Manager-4.3/job/${env.JOB_NAME}"
+    product_name = "SUSE-Manager-4.3"
+    short_product_name = "suma43"
```

Comment:
Be caution, have we domains for suma43-xxxx ? Or we only have domains for suma-xxx ?

Reply:
This variable is used to reference a specific url where repos are created. Those repos are specific for suma4.3, I think makes sense to have the complete nameversion. Another story is that we should refactor the uyuni-pr to be named uyuni-pr instead of suma-pr, but that is another refactor/task I would mix up here.